weatherbot/puller.py

```diff
@@ -7,3 +7,7 @@ def get_new(updates, update_id):
         if should_add(item, update_id):
             buffer.append(item)
     return buffer
+
+def add_queue(queue, buffer):
+    queue.append(buffer)
+    return queue
```

Comment:
This is something you haven't seen yet. But you don't need to return the queue, because we mutated it. "Owner" of the queue will see the changes without returning it.

In other words, this function should just mutate the queue by adding elements and return nothing.